xrpl/models/ledger_objects/nftoken_offer.py

```diff
@@ -19,19 +19,71 @@ class NFTokenOffer(LedgerObject):
     """The model for the `NFTokenOffer` Ledger Object"""
 
     amount: Union[str, NFToken] = REQUIRED  # type: ignore
-    flags: Union[int, NFTokenOfferFlags] = REQUIRED  # type: ignore
```

Comment:
The `NFTokenOfferFlags` has not been used anywhere else in the code

Reply:
Like Mayukha said, a class doesn't need to be used in other parts of code.